translations/monero-core_nl.ts

```diff
@@ -580,217 +553,212 @@ S Send, R Receive, H History, D Advanced, E Settings.</translatorcomment>
         <translation>Annuleren</translation>
     </message>
     <message>
-        <location filename="../components/NewPasswordDialog.qml" line="244"/>
+        <location filename="../components/NewPasswordDialog.qml" line="237"/>
         <source>Continue</source>
         <translation>Doorgaan</translation>
     </message>
 </context>
 <context>
     <name>PasswordDialog</name>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="88"/>
+        <location filename="../components/PasswordDialog.qml" line="86"/>
         <source>Please enter wallet password</source>
         <translation>Vul het wachtwoord voor de portemonnee in</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="88"/>
-        <source>Please enter wallet password for:&lt;br&gt;</source>
-        <translation>Vul het wachtwoord van de portemonnee in voor:&lt;br&gt;</translation>
+        <location filename="../components/PasswordDialog.qml" line="86"/>
+        <source>Please enter wallet password for: </source>
+        <translation>Vul het wachtwoord in voor de portemonnee: </translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="156"/>
+        <location filename="../components/PasswordDialog.qml" line="150"/>
         <source>Cancel</source>
         <translation>Annuleren</translation>
     </message>
     <message>
-        <location filename="../components/PasswordDialog.qml" line="169"/>
+        <location filename="../components/PasswordDialog.qml" line="161"/>
         <source>Continue</source>
         <translation>Doorgaan</translation>
     </message>
 </context>
 <context>
     <name>PrivacyLevelSmall</name>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="100"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="96"/>
         <source>Low</source>
         <translation>Laag</translation>
     </message>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="109"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="105"/>
         <source>Medium</source>
         <translation>Gemiddeld</translation>
     </message>
     <message>
-        <location filename="../components/PrivacyLevelSmall.qml" line="118"/>
+        <location filename="../components/PrivacyLevelSmall.qml" line="114"/>
         <source>High</source>
         <translation>Hoog</translation>
     </message>
 </context>
 <context>
     <name>ProgressBar</name>
     <message>
-        <location filename="../components/ProgressBar.qml" line="41"/>
-        <source>Establishing connection...</source>
-        <translation>Bezig met verbinding te maken...</translation>
+        <location filename="../components/ProgressBar.qml" line="38"/>
+        <source>%1 blocks remaining: </source>
+        <translation>Blokken te gaan in %1: </translation>
```

Comment:
Wellicht: `%1 Blokken te gaan:`

Reply:
That's what I thought at first, but if you look at the ProgressBar code, you'll see that the placeholders %1 stands for the syncType: Wallet/Daemon. 
```
    property string syncType // Wallet or Daemon
    property string syncText: qsTr("%1 blocks remaining: ").arg(syncType)
```
The number of remaining blocks is in the var `remaining` IIUC.